### PR TITLE
support rocksdb info log dir

### PR DESCRIPTION
**Pull request by siddontang**
We can use `info-log-dir` to let RocksDB save its log to other place, not in origin data storage path. 
But we should know that the log name is still LOG. 

@andelf @BusyJay @zhangjinpeng1987

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -325,6 +325,13 @@ fn get_rocksdb_db_option(config: &toml::Value) -> RocksdbOptions {
         get_toml_int(config, "rocksdb.compaction-readahead-size", Some(0));
     opts.set_compaction_readahead_size(compaction_readahead_size as u64);
 
+    let info_log_dir = get_toml_string(config, "rocksdb.info-log-dir", Some("".to_owned()));
+    if !info_log_dir.is_empty() {
+        opts.create_info_log(&info_log_dir).unwrap_or_else(|e| {
```
Comment (zhangjinpeng1987) [inline src/bin/tikv-server.rs]: It is better to set `log_file_time_to_roll ` or `max_log_file_size ` rotate the LOG.

**Comment by siddontang** [comment]
PTAL @zhangjinpeng1987

**Comment by siddontang** [comment]
PTAL @BusyJay @andelf

File: etc/config-template.toml [hunk 2]
```diff
@@ -99,6 +99,13 @@ max-peer-down-duration = "5m"
 endpoints = ""
 
 [rocksdb]
+# RocksDB info log directory, if set, RocksDB will save the log in `info-log-dir/LOG`.
+# info-log-dir = ""
+# Maximal size of the info log, if new info log will be created if size is large than it. 
+# info-log-max-size = 0
```
Comment (BusyJay) [inline etc/config-template.toml]: Can this be set even when info-log-dir is empty?

**Comment by siddontang** [comment]
PTAL @BusyJay

File: etc/config-template.toml (hunk 2 above)
Comment (siddontang) [inline etc/config-template.toml]: yes

**Pull request merged**